demos/common/src/ie_config_helper.cpp

```diff
@@ -3,36 +3,49 @@
 //
 
 #include "samples/ie_config_helper.hpp"
+#include "samples/args_helper.hpp"
 
-void configureInferenceEngine(Core& ie,
-                              std::string& deviceString,
-                              std::string& deviceInfo,
-                              const std::string& lString,
-                              const std::string& cString,
-                              bool pc) {
-    for (char& ch : deviceString) {
-        ch = std::toupper(ch);
-    }
+#include <cldnn/cldnn_config.hpp>
+#include <vpu/myriad_config.hpp>
+
+void formatDeviceString(std::string& deviceString) {
+    bool preserveCase = false;
 
-    std::stringstream strBuffer;
-    strBuffer << ie.GetVersions(deviceString);
-    deviceInfo = strBuffer.str();
+    for (size_t i = 0; i < deviceString.size(); ++i) {
+        // These two conditions handle the special case of MYRIAD device names in "ma1234" format, where letters should
```

Comment:
https://devcloud.intel.com/edge/ does not report NCS names in `ie.GetAvailableDevices()`, but I found a name like this `ma2485` in pbsnodes and it looks similar to NCS names that I've seen before while working with real NSCes.

Reply:
`ie.GetAvailableDevices()` reports NCS names when there are more than one of them. You should try a set up with more than one stick. The name you've found looks correct, but it would be great to know if IE can take that name capitalized.